ee/docker-ee/ubuntu.md

```diff
@@ -1,5 +1,5 @@
 ---
-description: Instructions for installing Docker Engine - Enterprise on Ubuntu
+description: Instructions for installing Docker Enterprise on Ubuntu
```

Comment:
The Title for the page should be changed to reflect the revised description ("Installing Docker Enterprise on Ubuntu").